lambda-ssm-parameter-terraform/src/app.js

```diff
@@ -34,12 +37,12 @@ exports.handler = async (event, context) => {
         Overwrite: true,
         Type: "String",
       };
-      result = await ssm.putParameter(ssmPutParams).promise()
+      result = await ssmClient.send(new PutParameterCommand(ssmPutParams))
     } else if (method == "GET") {
       const ssmGetParams = {
         Name: parameterName,
       };
-      result = await ssm.getParameter(ssmGetParams).promise()
+      result = await ssmClient.send(new GetParameterCommand(ssmGetParams))
```

Comment:
note: Some error occurs. Because in Node.js 18 and later runtimes, AWS SDK v3 is bundled, so I rewrote it to v3. See articles below.

- https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/
- https://aws.amazon.com/blogs/compute/node-js-22-runtime-now-available-in-aws-lambda/